src/datasets/minecraft_savedata/mod.rs

```diff
@@ -19,6 +19,7 @@ use minecraft_savedata_prost as pb;
 #[cfg(feature = "nanoserde")]
 use nanoserde::{DeBin, SerBin};
 use rand::Rng;
+#[cfg(feature = "rkyv")]
 use rkyv::traits::NoUndef;
```

Comment:
Change the impl for `ArchivedGameType` to be `unsafe impl rkyv::traits::NoUndef for ArchivedGameType {}` and remove this `use`

Reply:
Done